shotover/src/transforms/kafka/sink_cluster/node.rs

```diff
@@ -8,6 +9,46 @@ use kafka_protocol::protocol::StrBytes;
 use std::time::Duration;
 use tokio::io::split;
 
+pub struct ConnectionFactory {
+    tls: Option<TlsConnector>,
+    connect_timeout: Duration,
+    handshake_message: Option<Message>,
```

Comment:
neither `handshake_message` or `auth_message` are set or read anywhere yet.
Can you explain their purpose?